Improve documentation v4 (LC15) (#4973)
Deletescape-Media/Lawnchair: README.md

    <img alt="Google Pixel running Lawnchair Launcher with green wallpaper" src="docs/assets/device-frame.png" width="250px">
</picture>

Lawnchair is a free, open-source home app for Android. Taking Launcher3—Android’s default home app—as a starting point, it ports Pixel Launcher features and introduces rich customization options.

> This branch houses the codebase of Lawnchair 15, which is currently in alpha and is based on Launcher3 from Android 15. For Lawnchair 9 to 14, see the branches with the `9-` to `14-` prefixes, respectively.

Reviewer: Once we're in beta, it doesn't really make sense to return back to alpha. I'd suggest changing this to `beta`, or more accurately, in `development` (as we haven't released a new version yet). We can change this to `beta` once we've released a new version.

## Features

-   **Material You Theming:** Adapts to your wallpaper and system theme.
-   **At a Glance Widget:** Get information *At a Glance* with support for [Smartspacer](https://github.com/KieronQuinn/Smartspacer) extensions.
-   **QuickSwitch Support:** Seamlessly integrate itself as recent apps provider (requires root on Android 10 and newer).
-   **Global Search:** Find apps, contacts, and web results quickly from your home screen.

> -   **Highly customizable:** Customize Icons, fonts, and colors to your liking

Reviewer: This doesn't seem to follow a parallel structure. Here's a suggestion to make it more consistent:

```suggestion
-   **Material You Theming:** Adapts to your wallpaper and system theme.
-   **At a Glance Widget:** Displays information *at a glance* with support for [Smartspacer](https://github.com/KieronQuinn/Smartspacer) extensions.
-   **QuickSwitch Support:** Integrates with Android Recents on Android 10 and newer. (requires root)
-   **Global Search:** Allows quick access to apps, contacts, and web results from the home screen.
-   **Customization Options:** Provides options to tweak icons, fonts, and colors to your liking.
```

I've also edited it to sound less like marketing. Such content would fit at our website instead.

      <source media="(prefers-color-scheme: dark)" srcset="docs/assets/badge-google-play.png" height="60">
      <img alt="Get it on Google Play" src="docs/assets/badge-google-play.png" height="60">
    </picture>
  </a>
  <a href="https://apt.izzysoft.de/fdroid/index/apk/app.lawnchair">
    <picture>
      <source media="(prefers-color-scheme: dark)" srcset="docs/assets/badge-izzyondroid.png" height="60">
      <img alt="Get it on IzzyOnDroid" src="docs/assets/badge-izzyondroid.png" height="60">
    </picture>
  </a>
  <a href="https://apps.obtainium.imranr.dev/redirect?r=obtainium://add/https://github.com/LawnchairLauncher/lawnchair/">
    <picture>
      <source media="(prefers-color-scheme: dark)" srcset="docs/assets/badge-obtainium.png" height="60">
      <img alt="Get it on Obtainium" src="docs/assets/badge-obtainium.png" height="60">
    </picture>
  </a>
    <a href="https://github.com/LawnchairLauncher/lawnchair/releases/latest">
    <picture>
      <source media="(prefers-color-scheme: dark)" srcset="docs/assets/badge-github.png" height="60">
      <img alt="Get it on GitHub" src="docs/assets/badge-github.png" height="60">
    </picture>
  </a>
</p>

> Lawnchair on the Play Store will install as a different app from other sources and features may be restricted to comply with the publishing rules.

Reviewer: ```suggestion
Lawnchair on Play Store will install as a different app from other sources. Sone features may be restricted to comply with Google Play's publishing rules.
```

### Development builds

> Interested in keeping yourself up-to-date with every Lawnchair development? Try our development build, these builds offer the latest features and bug fixes at a cost of being slower and *might* introduce new bugs that have never been seen in stable. If you'd like to join in the fun, don't forget to make backup(s) and proceed with cautions.

Reviewer: ```suggestion
Interested in keeping yourself up-to-date with every Lawnchair development? Try our development build!

These builds offer the latest features and bug fixes at a cost of being slower and introducing new bugs. Ensure that you make backups before installing!
```
